services/base/shared_libs/src/scanhub_libraries/models.py

```diff
@@ -231,8 +232,10 @@ class Config:
     description: str
     type: TaskType
     args: dict[str, str]
-    artifacts: dict[str, list[dict[str, str]]]
-    task_destinations: list[dict[str, str]]
+    artifacts: dict[str, str]
+    destinations: dict[str, str]
+    # artifacts: dict[str, list[dict[str, str]]]
+    # task_destinations: list[dict[str, str]]
```

Comment:
maybe better to remove when unused?

Reply:
You are right, we have it in the git history